modules/ROOT/pages/cloudhub-api.adoc

```diff
@@ -85,23 +85,44 @@ Here is an example of data received in JSON format in response to a request to g
 [source,json,linenums]
 ----
 {
-  "domain":"hello",
-  "fullDomain":"hello.cloudhub.io",
-  "workers":1,
-  "hasFile":false,
-  "muleVersion": "3.1.2",
+ "versionId": "xxxxxxxx",
+  "domain": "hello",
+  "fullDomain": "hello.cloudhub.io",
```

Comment:
Let's use a newer domain like hello.us-e1.cloudhub.io.
That domain is now deprecated
```suggestion
  "fullDomain": "hello.us-e1.cloudhub.io",
```